config/routes.rb

```diff
@@ -4,7 +4,11 @@
   # Defines the root path route ("/")
   root "users#index"
   get "/users/:user_id/posts", to: "posts#index", as: "user_posts"
+  get "/users/:user_id/posts/create", to: "posts#new", as: "new_post"
+  post "/users/:user_id/posts/create", to: "posts#create", as: "create_post"
   get "/users/:user_id/posts/:post_id", to: "posts#show", as: "user_post"
+  post "/users/:user_id/posts/:post_id", to: "posts#create_comment", as: "create_comment"
+  get "/users/:user_id/posts/:post_id/like", to: "posts#add_like", as: "add_like"
```

Comment:
- [ ]  When you create the `likes_controller`, make sure you update your `routes` accordingly.